tuxemon/states/combat/combat.py

```diff
@@ -151,10 +151,12 @@ def compute_text_animation_time(message: str) -> float:
 
 class TechniqueAnimationCache:
     def __init__(self) -> None:
-        self._sprites: Dict[Union[Technique, Condition], Optional[Sprite]] = {}
+        self._sprites: Dict[
+            Union[Technique, Condition, Item], Optional[Sprite]
```

Comment:
This type looks complex and a part of it is repeated quite a lot of times in this file.

I think we should extract `Union[Technique, Condition, Item]` as a type alias at the top of the file.
I'm just not sure about what's the common thing between Technique, Condition and Item to name this type properly, but I trust you to find the perfect name! :D

Reply:
Yeah, I take note, but it'll be applied in another PR.